landing_page/static/landing_page/js/movie.js

```diff
@@ -1,44 +1,86 @@
-
-// Buttons for the movie page's 
+// Buttons for the movie page's
 // context change depending on button clicked, testing visuals
-function movieOverview(){
-    document.getElementById("movie_summary").innerHTML = 
-    `Description of <b>Dark Knight</b> Movie: Movie Summary/Plot
-    `;
+function movieOverview(movie_data) {
+  console.log("movie_data: ", movie_data);
+  streaming = JSON.parse(movie_data.watch_providers);
+
+  if (streaming.length == 0) {
+    streaming = "No streaming options available...";
+  } else {
+    streaming = streaming.toString();
+  }
+  document.getElementById(
+    "movie_summary"
+  ).innerHTML = `<p><strong>Description:</strong> </p>
+      <p> ${movie_data.overview} </p>
+      <div id="streaming_info">
+        <strong>Streaming:</strong> <p>${streaming}</p>
+      </div>`;
 }
-function movieDetails(){
-    document.getElementById("movie_summary").innerHTML = 
-    `<div>Director(s):
-    </div>
-    <div> 
-    Writer(s):
-    </div>
-    <div>
-    Star(s):
-    </div>
-    <div>
-    Genre(s):
-    </div>
-    `;
+
+function movieDetails(movie_data) {
+  movie_director = JSON.parse(movie_data.director);
+  movie_director = movie_director.toString();
+
+  movie_genres = JSON.parse(movie_data.genres);
+  movie_genres = movie_genres.toString();
```

Comment:
the movie genres don't have spaces between them - let's fix that!
![image](https://github.com/jonathanhouge/ReeldIn/assets/111781413/5af99c9a-18eb-4fe2-b1a2-751a88e17639)

Reply:
this should still be fixed